.readthedocs.yml

```diff
@@ -1,7 +1,10 @@
 version: 2
 
 sphinx:
-  fail_on_warning: true
+  # This should not fail on errors.
+  # We have a separate build process which fails on errors and shows all
+  # the errors in one go.
+  fail_on_warning: false
```

Comment:
I think it's worth enabling this once we can ignore the warnings in ToC https://github.com/sphinx-doc/sphinx/pull/8289

(Sphinx 3.3.0)

Reply:
Right now, it is blocked by a different issue.

When fail on warnings is True, read the docs will not build the HTML files..so you can't check the partial output.

I was expecting that this PR will fix it https://github.com/readthedocs/readthedocs.org/pull/7251 

but when sphinx build fails, I still don't get HTML files